pkgs/development/python-modules/secure/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildPythonPackage, fetchFromGitHub
+, maya
+, requests
+}:
+
+buildPythonPackage rec {
+  version = "0.2.1";
+  pname = "secure";
+
+  src = fetchFromGitHub {
+    owner = "typeerror";
+    repo = "secure.py";
+    rev = "v${version}";
+    sha256 = "1nbxwi0zccrha6js14ibd596kdi1wpqr7jgs442mqclw4b3f77q5";
+  };
+
+  propagatedBuildInputs = [ maya requests ];
+
+  # no tests in release
+  checkPhase =  ''
+    python -c 'import secure'
```

Comment:
pythonImportsCheck

Reply:
for some reason, that one was complaining about `not able to find setup.py`

Reply:
I think the test runner probably changed the directory i was in

Reply:
using `doCheck = false;` fixes it, although this should probably be fixed that a test runner which fails to collect tests changes the directory from sourceRoot